packages/embedex/README.md

```
@@ -0,0 +1,91 @@
+    *
+    * greet("world");
+    * ```
+    */
+   function greet(name: string) {
+     console.log(`Hello, ${name}!`);
+   }
+   ````
+
+## Reference
+
+```
```

Markdownlint

Fenced code blocks should have a language specified

(MD040, fenced-code-language)